### PR TITLE
bot mention commands & mongo fixes

### PR DESCRIPTION
**Pull request by Vycton**
- Added the option to send a command by starting a message with the bot's @handle.
- Improved error handling of persistence
- Updated the README to reduce risk of errors occuring when following the 'Run your own bot server' instructions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,8 +12,9 @@ The easiest way to get Omni running on your server is by using Docker.
 1. You will first have make your own bot account on the [Discord Developer Portal](https://discord.com/developers/). Instructions for doing this are available [here](https://discordpy.readthedocs.io/en/latest/discord.html).
 2. Download Docker from [docker.com](https://www.docker.com/) and install it on your system.
 3. Now, you have to spin up a MongoDB container which will manage Omni's storage. Make a new directory where you want MongoDB to store its files.
-4. Use ```docker run -d -v /path/to/your/directory:/data/db mongo``` to spin up a MongoDB container.
-5. Use ```docker run mbreemhaar/omni:latest your-bot-token``` to spin up an Omni container. This container should now automatically connect to the MongoDB container.
+4. Use ```docker run --name omni_mongo -d -p 27017:27017 -v /path/to/your/directory:/data/db mongo``` to spin up a MongoDB container.
+5. Use ```docker inspect -f '{{range.NetworkSettings.Networks}}{{.IPAddress}}{{end}}' omni_mongo``` to obtain the IP address of the mongo container
```
Comment (mbreemhaar) [inline README.md:16]: Using these instructions, the Mongo container should always be on localhost.

File: README.md [hunk 2]
```diff
@@ -12,8 +12,9 @@ The easiest way to get Omni running on your server is by using Docker.
...
+6. Use ```docker run --name omni -e DB_IP='IP-from-previous-step' mbreemhaar/omni:latest your-bot-token``` to spin up an Omni container. This container should now automatically connect to the MongoDB container.
```
Comment (mbreemhaar) [inline README.md:17]: ```DB_IP``` is set to ```localhost``` by default so it does not have to be set here.

File: omni.py [hunk 3]
```diff
@@ -138,19 +138,28 @@ async def on_message(message):
         return
     
     prefix = persistence.get_prefix(message.guild)
-    if message.content.startswith(prefix):
+    mention = "<@!{}>".format(client.user.id)
```
Comment (mbreemhaar) [inline omni.py:141]: I think we could use some comments here...

**Review by mbreemhaar — CHANGES_REQUESTED** [review]
Maybe it's also a good idea to make a 'getPrefix' command as described in #10 (Additional context)

File: README.md (hunk 1 above)
Comment (Vycton) [reply README.md:16]: It was not for me when using windows WSL. The updated instructions should work regardless of the container's IP address.

File: README.md (hunk 2 above)
Comment (Vycton) [reply README.md:17]: Unless inspect returns a different IP address, as was the case for me in windows WSL. Might it be better to include all of these commands (with different cases) in a script, and have it read the user's preferred options from a configuration file?

File: omni.py (hunk 3 above)
Comment (Vycton) [reply omni.py:141]: Will do

**Comment by Vycton** [comment]
> 
> 
> Maybe it's also a good idea to make a 'getPrefix' command as described in #10 (Additional context)

I've added this

**Review by mbreemhaar — APPROVED** [review]

**Pull request merged**